config/locales/en.yml

```
@@ -1057,7 +1058,7 @@ en:
         order: "Order By"
         rating: "Rating"
         created_at: "Created At"
-        stars: "Stars"
+        min_stars: "Minimum amount of stars"
```

I think stars are countable:
```suggestion
        min_stars: "Minimum number of stars"
```